### ISSUE TITLE
tarantoolctl rebootstrap 

### ISSUE THREAD

**Issue by rtsisyk** [tarantool/tarantool#2908]
Add a command to tarantoolctl to rebootstrap replica from remote master, i.e. remove all data and boostrap replica from the remote master.

**Comment by kostja** [comment]
This is already reflected in a more complicated issue, https://github.com/tarantool/tarantool/issues/461
tarantoolctl is a quick fix. Whoever does this ticket, let's not forget to clean the old replica's uuid from the master as soon as we rebootstrap.

Re name of the command: we should perhaps call it "bootstrap" instead of rebootstrap, since it will be usable both for bootstrap and rebootstrap.

**Comment by rtsisyk** [comment]
Blocked by https://github.com/tarantool/tarantool/issues/2967

**Comment by kostja** [comment]
Won't fix - an instance automatically rebootstraps itself in the patch for gh-461 which is in review.
